### DISCUSSION TITLE
What kind of projects are you working on?

### DISCUSSION THREAD

**Discussion by fossegutten** [17cupsofcoffee/tetra#239] [General]
With the 0.6 release, it looks like Tetra now has most of the stuff I need for making a simple platformer game. 
I made some before, in other languages, with mixed results. Mostly I had issues with too much functionality I didnt need in other engines, or compiling issues in C/C++. Hope to become good enough in Rust to be able to set up collisions and a map importer or editor, because the tools are awesome.

What kind of games are you making in tetra?

**Comment by 17cupsofcoffee** [comment]
Hopefully some other people will chime in, but I'll answer too :)

My main project I'm tinkering with at the moment is a [Fire Emblem style game](https://twitter.com/17cupsofcoffee/status/1352024256067022848), which has been interesting - I feel like turn-based games require you to be a lot more strict with seperating game logic/rendering logic, and it's taken me quite a few tries to arrive at an architecture that feels (relatively) clean.

I think this would be a pretty huge project to do if I wanted to do it well, though - hoping to do some smaller games/game jams this year rather than never getting anything released :)
